src/Runner.Worker/JobRunner.cs

```diff
@@ -40,21 +39,34 @@ public async Task<TaskResult> RunAsync(Pipelines.AgentJobRequestMessage message,
             Trace.Info("Job ID {0}", message.JobId);
 
             DateTime jobStartTimeUtc = DateTime.UtcNow;
+            IJobServer jobServer = null;
+            IRunServer runServer = null;
 
             ServiceEndpoint systemConnection = message.Resources.Endpoints.Single(x => string.Equals(x.Name, WellKnownServiceEndpointNames.SystemVssConnection, StringComparison.OrdinalIgnoreCase));
+            if (string.Equals(message.MessageType, JobRequestMessageTypes.RunnerJobRequest, StringComparison.OrdinalIgnoreCase))
+            {
+                runServer = HostContext.GetService<IRunServer>();
```

Comment:
yes we are not starting `_jobServerQueue` here, that's the one that drives sending updates, that's all out of scope and will come in later.